discovery_engine/pubspec.yaml

```diff
@@ -8,27 +8,27 @@ environment:
   sdk: '>=2.14.2 <3.0.0'
 
 dependencies:
-  crypto: 3.0.1
-  equatable: 2.0.3
-  freezed_annotation: 0.15.0
-  hive: 2.0.4
-  http: 0.13.4
-  http_retry: 0.2.0
-  json_annotation: 4.3.0
-  logger: 1.1.0
-  meta: 1.7.0
-  universal_platform: 1.0.0+1
-  uuid: 3.0.5
+  crypto: ^3.0.1
+  equatable: ^2.0.3
+  freezed_annotation: ^1.1.0
+  hive: ^2.0.5
+  http: ^0.13.4
+  http_retry: ^0.2.0
+  json_annotation: ^4.4.0
+  logger: ^1.1.0
+  meta: ^1.7.0
+  universal_platform: ^1.0.0+1
+  uuid: ^3.0.5
   async_bindgen_dart_utils:
     git:
       url: https://github.com/xaynetwork/xayn_async_bindgen.git
       path: async_bindgen_dart_utils
```

Comment:
maybe we should specify `ref` as well, so we have more controll over the version used, wdyt?